framework/templates/ecl-filters/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@ec-europa/ecl-filters",
+  "version": "0.3.1",
+  "description": "Filter a list of elements",
+  "main": "n/a",
+  "style": "n/a",
+  "author": "European Commission",
+  "license": "EUPL-1.1",
+  "dependencies": {
+    "@ec-europa/ecl-base": "^0.6.0",
+    "@ec-europa/ecl-buttons": "^0.7.1",
+    "@ec-europa/ecl-forms-checkboxes": "^0.5.1",
+    "@ec-europa/ecl-datepickers": "^0.5.1",
+    "@ec-europa/ecl-forms-form-groups": "^0.4.0",
+    "@ec-europa/ecl-forms-labels": "^0.5.0",
+    "@ec-europa/ecl-forms-selects": "^0.7.2",
+    "@ec-europa/ecl-forms-text-inputs": "^0.5.2",
+    "@ec-europa/ecl-grid": "^0.3.0",
+    "@ec-europa/ecl-margin": "^0.3.0",
+    "@ec-europa/ecl-tags": "^0.4.1",
+    "@ec-europa/ecl-typography-headings": "^0.5.0"
+  },
+  "publishConfig": {
```

Comment:
Just out of curiosity - why publishing this package? It's something to be used only in the framework as a demo in the style guide no? If a user wants to take package and build only a search with filters, then maybe we should provide an optional entry css file?